examples/pipelines/ccfraud/submit.py

```diff
@@ -241,7 +241,7 @@ def fl_ccfraud_basic():
                     mode=silo_config.testing_data.mode,
                     path=silo_config.testing_data.path + f"/test.csv",
                 ),
-                metrics_prefix=silo_config.compute,
+                metrics_prefix=silo_config.computes[0],
```

Comment:
@garg-amit when did we change schema to include multiple computes and why is this necessary?

Reply:
in our config, we do not have "compute" anymore but "computes" for each silo. This change simply avoids the error report

Reply:
The multiple computes changes were introduced in the Jan's release https://github.com/Azure-Samples/azure-ml-federated-learning/releases/tag/v0.4.0

Reply:
the goal here was to have cpu+gpu for each pipeline, cpu for processing, gpu for training